ciphey/basemods/Decoders/galactic.py

```diff
@@ -0,0 +1,98 @@
+# community
+# by https://github.com/lukasgabriel
+
+from typing import Optional, Dict, List
+
+from ciphey.iface import Config, ParamSpec, T, U, Decoder, registry
+
+
+@registry.register
+class Galactic(Decoder[str, str]):
+    def decode(self, ctext: T) -> Optional[U]:
+        """
+        Takes a string written in the 'Standard Galactic Alphabet' 
+        (aka Minecraft Enchanting Table Symbols) and translates it to ASCII text.
+        """
+
+        result = ""
+        galactic_letters = [
+            "ᔑ",
+            "ʖ",
+            "ᓵ",
+            "↸",
+            "ᒷ",
+            "⎓",
+            "⊣",
+            "⍑",
+            "╎",
+            "⋮",
+            "ꖌ",
+            "ꖎ",
+            "ᒲ",
+            "リ",
+            "𝙹",
+            "!",
+            "ᑑ",
+            "∷",
+            "ᓭ",
+            "ℸ",
+            "⚍",
+            "⍊",
+            "∴",
+            "|",
+            "⨅",
+        ]
+        letters = list("abcdefghijklmnopqrstuvwyz")
+        galactic_dict = {galactic_letters[i]: letters[i] for i in range(25)}
+
+        # Ensure that ciphertext is a string
```

Comment:
I don't think you need to do this? Or you shouldn't? Either way, this seems like a search problem (searcher shouldn't give it to this if it isn't a string, I think). Is this in my documentation? I might need to explore this some more.

@Cyclic3

Reply:
Took the whole block out, left it in when I copied over the template from atbash but I already suspected that both the case conversion and the type checking was not necessary.